mailgun/route_handlers.py

```diff
@@ -69,3 +69,64 @@ def handle_mailing_list_email_route(request):
         ml.send_mail(sender, subject, html=content)
 
     return JsonResponse({'success': True})
+
+
+
+
+@csrf_exempt
+@authenticate()
+@require_http_methods(['POST'])
+def handle_outgoing_emails_route(request):
+    """
+    Handles the Mailgun route action when email is sent to any Mailgun mailing list.
+    1. Check the access level for the mailing list to see if it is set to 'staff'
+    2. Verify if user can post to the list: If access level of mailing list is 'staff' and sender is not a  course staff
+     member, the mail should not be sent
+
+    :param request:
+    :return:
+    """
+    sender = request.POST.get('sender')
+    recipient = request.POST.get('recipient')
+    subject = request.POST.get('subject')
+    message_body = request.POST.get('body-plain')
+    logger.info("Handling Mailgun outgoing mailing list email from %s to %s", sender, recipient)
+
+    # fetch the mailing list access level
+    mailing_list_access_level = None
```

Comment:
Could you reference ml.access_level below instead of creating this extra variable?